src/components/Common/SearchInput.tsx

```diff
@@ -183,7 +185,7 @@ const SearchByMultipleFields: React.FC<SearchByMultipleFieldsProps> = ({
     }
   }, [searchValue]);
 
-  const renderSearchInput = useMemo(() => {
+  const renderInput = useMemo(() => {
```

Comment:
i'd recommend keeping this component defined outside; code is just uglier when component is inlined inside another component;